assets/assets_identifiers_ws.py

```diff
@@ -0,0 +1,54 @@
+import json
+import asyncio
+import pieces_os_client as pos_client
+import websockets
+from pieces import config 
+import threading
+
+class AssetsIdentifiersWS:
+    def __init__(self, on_message_callback):
+        self.ws = None
+        self.is_connected = False
+        self.on_message_callback = on_message_callback
+
+        # Create a new event loop
+        self.loop = asyncio.new_event_loop()
+
+        # Run the event loop in a new thread
+        self.thread = threading.Thread(target=self.start_event_loop, args=(self.loop,))
+        self.thread.start()
+
+        # Open the websocket connection
+        asyncio.run_coroutine_threadsafe(self.open_websocket(), self.loop)
```

Comment:
what is the purpose of using this within an event loop?

Reply:
To keep the websocket alive